src/_includes/partials/components/filters.njk

```diff
@@ -0,0 +1,83 @@
+{% set filterableOptions = { projects: [], types: [], topics: [], publishers: [] } %}
+{% for resource in collections['resources_' + lang] %}
+    {% set filterableOptions = filterableOptions | objectArrayPush("projects", resource.data.projects) %}
+    {% set filterableOptions = filterableOptions | objectArrayPush("types", resource.data.type) %}
+    {% set filterableOptions = filterableOptions | objectArrayPush("topics", resource.data.topics) %}
+    {% set filterableOptions = filterableOptions | objectArrayPush("publishers", resource.data.publisher) %}
+{% endfor %}
+<div id="filters" class="filters flow">
+    <h3>{% __ 'filters' %}</h3>
+    <div class="flow">
+        <fieldset>
```

Comment:
We should check to make sure that there are values for all of these and only show filters where we have some to apply that have an effect. E.g. right now there are no topics and only one type; I would suggest that we hide both topics and types until we have more than one of each available.